src/main/java/duke/Task.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+public class Task {
+
+    //variables
+    protected String description;
+    private boolean isDone;
+
+    //constructor
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    //getters
+    public String getDescription() {
+        return description;
+    }
+    public boolean getDone() {
+        return isDone;
+    }
+
+    //setters
+    public void setDone() {
+        this.isDone = true;
+    }
```

Comment:
Suggest to change to getIsDone() and setIsDone() respectively, since you are modifying the variable isDone.